### DISCUSSION TITLE
Insights -> Traffic seems to show my own views

### DISCUSSION THREAD

**Discussion by metablaster** [community/community#41710] [Discussions] [answered]
### Select Topic Area

Question

### Body

There already are questions and answers on this but it seems this is not entirely correct or there was some change, for ex:
[Does the GitHub traffic graph include your own views?](https://stackoverflow.com/questions/22839597/does-the-github-traffic-graph-include-your-own-views)
and [Traffic under Insight of repository #23048](https://github.com/orgs/community/discussions/23048)

Under `Insights -> Traffic` when you scroll down in your repo I see this:

![](https://i.imgur.com/xZwvGOo.png)

It says 43 views of "Traffic" site, while in fact I'm the only one able to see this page, if I paste the link into "private" browser window it asks me for login, so there is no way anyone could visit this page to be counted.

I'm logged into github all the time and do not clear github cookies nor do I block any content from what I see, (I have [uBlock origin][ublock] and [CAD][cad] installed btw.)

So if github doesn't count my own views while logged in, then how is it possible that view of the "traffic" is not 0?
I also see high amount of views compared to unique visits which is very likely me because I often read my own docs and diffs and return to branch page ex. after reading notifications, which it seems counts as a new page view.

It's interesting though if I hit `CTRL + F5` to clean reload page it doesn't count it as page view, but also if visit my repo from "private" browser window it does not count it as unique view, but I assume this is because the graph isn't instantly updated?
therefore it's difficult to test.

So ultimately, still in 2022, does github track your own views or not?

**EDIT:**

Insights page options from private browser window does not show an option to view traffic, the following options are available to the visitor:

![](https://i.imgur.com/HucxJhK.png)

When you click on `Insights` it default to `Pulse` page.

I can see traffic per individual pages on this screenshot, but "Traffic" is available only if you  log in, for ex:

![](https://i.imgur.com/m31eMWC.png)

Screenshot below shows individual page counts where traffic is one of them, there is no `Insights` since it defaults to pulse, therefore those `Traffic` counts if they're not mine should be `Pulse` counts which is default page for `Insights`

![](https://i.imgur.com/7tRpHew.png)

[cad]: https://github.com/Cookie-AutoDelete/Cookie-AutoDelete
[ublock]: https://github.com/gorhill/uBlock

**Comment by radar-blip** [comment, marked as answer]
Your own views are not counted while you are logged in.

The marked answer here  #23048 is probably the most accurate you will find.

**Reply by metablaster** [reply]
If that's correct then why does my screenshot above show 43 views on page which only I can access?

**Reply by radar-blip** [reply]
Are you talking about a private repository? Bots maybe?

**Reply by metablaster** [reply]
it's not private repo, only the "Traffic" portion is private, nobody can view traffic of your repo except you, therefore even bots should not be able to see your traffic. but for some reason there are views, and the only logical conclusion is that github tracks your own views or there is some other explanation.

**Reply by radar-blip** [reply]
Its the repository itself that is being viewed, not the actual traffic page. Does that make sense?

**Reply by metablaster** [reply]
That's one possibility to conclude, that only only traffic page counts your own views, but it doesn't make sense to me, traffic page should not be on the list at all if your views aren't counted, it only brings suspition that the rest counts your views as well.

**Reply by radar-blip** [reply]
It has nothing to do with your own views, its other people viewing your repository.

If I was to look at one of your repositories, it would be counted as 1 view (on your traffic tab).
So, those 43 views are from other people.

**Reply by metablaster** [reply]
This should read as `Pulse` view count then not `Traffic` since "traffic" isn't visible under "insights", I've updated my question with screenshots, it's impossible to count "insights" because it defaults to `Pulse` page when you click on "insights", for `Traffic` you need to explicitly click on it and only then it can count as page view. I'm sorry but this logically makes not sense.
